packages/rich-text/src/formatters/test/italic.tsx

```diff
@@ -14,15 +14,22 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import type { EditorState } from 'draft-js';
+import type { OrderedSet } from 'immutable';
+
 /**
  * Internal dependencies
  */
 import {
-  togglePrefixStyle,
   getPrefixStylesInSelection,
+  togglePrefixStyle,
 } from '../../styleManipulation';
-import { NONE, ITALIC } from '../../customConstants';
+import { ITALIC, NONE } from '../../customConstants';
```

Comment:
You're using a plugin to alphabetically sort imports, or?

Reply:
Not sure how this one happened